city_scrapers/spiders/chi_ssa_52.py

```diff
@@ -0,0 +1,168 @@
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+            }
+
+        if len(meet) == 6:
+            day_week = meet[0]
+            month = meet[1]
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            info = " ".join(meet[4:])
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+                "info": info,
+            }
+
+        return payload
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        item = item.replace("\xa0", "").strip()
+        return item
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        months = [
+            "JANUARY",
+            "FEBRUARY",
+            "MARCH",
+            "APRIL",
+            "MAY",
+            "JUNE",
+            "JULY",
+            "AUGUST",
+            "SEPTEMBER",
+            "OCTOBER",
+            "NOVEMBER",
+            "DECEMBER",
+        ]
+
+        try:
+            date = datetime.strptime(
+                f"{item['month_day']} {item['month']}, {item['year']}", "%d %B, %Y"
+            )
+        except ValueError:
+            for month in months:
+                ratio = SequenceMatcher(None, month, item["month"]).ratio()
+                if ratio > 0.5:
+                    date = datetime.strptime(
+                        f"{item['month_day']} {month}, {item['year']}", "%d %B, %Y"
+                    )
+
+        time = item["time"].split(":")  # [0] = hours, [1] = minutes
+        date = date.replace(hour=int(time[0]), minute=int(time[1]))
+        return date
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        return {
+            "address": "",
+            "name": "",
+        }
+
+    def _parse_links(self, item):
+        """Parse or generate links."""
+        return [{"href": "", "title": ""}]
```

Comment:
More info [in the docs](https://cityscrapers.org/docs/development/#links), but we'll want this to return an empty list rather than a list with one dict with empty values if we don't have any links to parse

Reply:
Thanks for the info. I'll change it